Allow testing fields via callbacks (#6040)
laravel-filament/filament: packages/forms/docs/07-testing.md

});
```

> You may pass a closure as a second argument in order to assert that a field passes a given "truth test". This is useful for asserting that a field has a specific configuration:

Reviewer: ```suggestion
You may pass a function as an additional argument in order to assert that a field passes a given "truth test". This is useful for asserting that a field has a specific configuration:
```

```php
use function Pest\Livewire\livewire;

it('has a title field', function () {
    livewire(CreatePost::class)

>         ->assertFormFieldExists('title', function (TextInput $input) {

Reviewer: ```suggestion
        ->assertFormFieldExists('title', function (TextInput $field): bool {
```

            return $input->isDisabled();
        });
});
```

> > Note that if you have multiple forms on a Livewire component, you can specify which form you want to check for the existence of the field like `assertFormFieldExists('title', 'createPostForm')`. In this case, you can also pass a closure as the third argument like `assertFormFieldExists('title', 'createPostForm', function (TextInput $input) { ... })`.

Reviewer: ```suggestion
> Note that if you have multiple forms on a Livewire component, you can specify which form you want to check for the existence of the field like `assertFormFieldExists('title', 'createPostForm')`.
```
